.env

```diff
@@ -0,0 +1,3 @@
+HOST=localhost
+PORT=8000
+DB_URL='mongodb+srv://bizimungu:0728038pp.PP@agakiriro.0xgn1.mongodb.net/blog?retryWrites=true'
```

Comment:
This file should not be on GitHub. They are your app's secrets, exposing them gives anyone on the internet to get into your DB and  do whatever they want there.

Reply:
Done